docs/release-notes/8.7.asciidoc

```diff
@@ -5,6 +5,12 @@
 [[release-notes-8.7.1]]
 === 8.7.1
 
+[discrete]
+[[known-issue-8.7.1]]
+==== Known issues
+
+* Index aliases and some data streams are not properly retrieved by the {elastic-sec} default data view.   
```

Comment:
Do you think we should add that it's fixed in 8.8?

Reply:
Discussed this with @yctercero and am resolving the thread since we don't typically include foward-looking statements in known issue summaries.